cmd/bridge/config/auth/authoptions.go

```
@@ -273,13 +276,19 @@ func (c *completedOptions) getAuthenticator(
 	var (
 		err                      error
 		userAuthOIDCIssuerURL    *url.URL
-		authLoginErrorEndpoint   = proxy.SingleJoiningSlash(baseURL.String(), server.AuthLoginErrorEndpoint)
-		authLoginSuccessEndpoint = proxy.SingleJoiningSlash(baseURL.String(), server.AuthLoginSuccessEndpoint)
+		authLoginErrorEndpoint   = proxy.SingleJoiningSlash(baseURL.Path, server.AuthLoginErrorEndpoint)
+		authLoginSuccessEndpoint = proxy.SingleJoiningSlash(baseURL.Path, server.AuthLoginSuccessEndpoint)
 		oidcClientSecret         = c.ClientSecret
```

golangci-lint (2.12.2)

[error] ineffectual assignment to oidcClientSecret

(ineffassign)